app/blueprints/fund_builder/routes.py

```diff
@@ -538,7 +543,9 @@ def create_export_files(round_id):
     generate_config_for_round(round_id)
     round_short_name = get_round_by_id(round_id).short_name
 
-    output_zip_path = create_export_zip(directory_to_zip=Config.TEMP_FILE_PATH / round_short_name, zip_file_name=round_short_name)
+    output_zip_path = create_export_zip(
```

Comment:
Shall we remove the formatting changes ?

Reply:
why....?

Reply:
I have seen narender also done the same formatting its ok it seems some was not done the precommit